bot.py

```diff
@@ -129,7 +129,8 @@ def in_script_dir(file_path):
 def get_envs():
     """Get required environment variables"""
     required_keys = (
-        'SLACK_ACCESS_TOKEN', 'BOT_ACCESS_TOKEN', 'GITHUB_ACCESS_TOKEN', 'SLACK_WEBHOOK_TOKEN', 'TIMEZONE',
+        'SLACK_ACCESS_TOKEN', 'BOT_ACCESS_TOKEN', 'GITHUB_ACCESS_TOKEN',
```

Comment:
Minor nitpick: one per line?  Feel free to ignore if you prefer.

Reply:
Sure